apps/chat-e2e/src/ui/webElements/folders.ts

```diff
@@ -383,22 +386,28 @@ export class Folders extends BaseElement {
   public getFolderEntityCheckbox(
     folderName: string,
     entityName: string,
-    index?: number,
+    folderIndex?: number,
+    entityIndex?: number,
```

Comment:
If we are counting from 1 and not from zero, we should start adding some check to ensure that the provided entityIndex is valid (at least not zero?)

Reply:
I guess this is responsibility of test author to use a valid index. Do you mean I should add smth like this?
`const index = entityIndex ? entityIndex === 0 ? 0 : entityIndex - 1 : 0;`

Reply:
throw an error in case of zero index